src/utils/hooks/Base64.js

```diff
@@ -1,3 +1,3 @@
-export const decodeBase64 = (base64) => new Buffer(base64, "base64").toString('ascii')
+export const decodeBase64 = (base64) => Buffer.from(base64, "base64").toString('ascii')
 
-export const encodeBase64 = (string) => new Buffer(string).toString('base64')
\ No newline at end of file
+export const encodeBase64 = (string) => Buffer.from(string).toString('base64')
```

Comment:
Pytanie, czy to jest potrzebne, skoro są funkcje btoa i tak dalej. Nie pamiętam dokładnie, ale wydaje mi się, że one miały jakieś ograniczenia, te ponoć są lepsze, ale mogło mi się powalić

Reply:
Pan WebStorm mówił mi raz, że `btoa` jest deprecated (chociaż teraz już nie mówi), więc to chyba lepsze